docs/buildah-bud.md

```diff
@@ -16,8 +16,8 @@ to a temporary location.
 
 **--authfile** *path*
 
-Path of the authentication file. Default is ${XDG_RUNTIME_DIR}/containers/auth.json
-which is set using `kpod login`
+Path of the authentication file. Default is ${XDG_RUNTIME\_DIR}/containers/auth.json, which is set using `kpod login`.
```

Comment:
is the \_DIR correct?

Reply:
Hmm, md ate up my slash.  Is RUNTIME(backslash)_DIR correct?  It appears to resolve ok, but I don't think the backslash is necessary

Reply:
I agree.

Reply:
Same comment in the other spots.

Reply:
in mark down when you put a word between underscores (_word_), it gets italicized. The back slash is to escape the _, so that doesn't happen.

Reply:
Gotta luv md!  Thanks for the info, something new learned today!